src/main/java/com/skiwi/githubhooksechatservice/mvc/beans/tasks/CommentsScanTask.java

```diff
@@ -15,13 +17,13 @@
 import com.skiwi.githubhooksechatservice.stackapi.StackExchangeComment;
 
 public class CommentsScanTask implements Runnable {
-    private static final Logger logger = LogManager.getLogger(CommentsScanTask.class);
-    
-    private Instant nextFetch = Instant.now();
-    private long lastComment;
-    private long fromDate;
-    private int remainingQuota;
-    
+	private static final Logger logger = LogManager.getLogger(CommentsScanTask.class);
+	
+	private Instant nextFetch = Instant.now();
+	private long lastComment;
+	private long fromDate;
+	private int remainingQuota;
+	
```

Comment:
@Vogel612 I actually asked what @Zomis preferred in chat, and he said whatever. I have them as tabs because half the time it was spaces, half the time it was tabs, and I personally prefer tabs.